contracts/exchange/lib/LibExchangeAccess.sol

```diff
@@ -17,6 +17,8 @@ library LibExchangeAccess {
     LibAddressSet.Data memberSet;
     /// @dev This is where we keep track of our member FAST memberships.
     mapping(address => LibAddressSet.Data) fastMemberships;
+    /// @dev This is where we keep track of our deactivated memberships.
+    LibAddressSet.Data deactivatedMemberSet;
```

Comment:
I like that you chose a `Set` rather than a `mapping(address => bool)`, as it gives us the ability to query the list of deactivated members.